api/soap/mc_filter_api.php

```diff
@@ -63,6 +63,7 @@
     'end_year' => FILTER_PROPERTY_DATE_SUBMITTED_END_YEAR,
     'tag_string' => FILTER_PROPERTY_TAG_STRING,
     'tag_select' => FILTER_PROPERTY_TAG_SELECT,
+    'alias' => FILTER_PROPERTY_ALIAS,
```

Comment:
I wouldn't expose the alias here.  Again that would be an internal operation that can be done internally.